src/js/lib/monster.ui.js

```diff
@@ -275,6 +275,28 @@ define(function(require) {
 				return monster.ui.getTemplatePhoneNumber(number.toString());
 			},
 
+			svgIcon: function(id, options) {
+				var template = monster.ui.getSvgIconTemplate({
+					id: id,
+					attributes: options.hash
+				});
+
+				return new Handlebars.SafeString(template);
+			},
+
+			telicon: function(id, options) {
```

Comment:
Variable declarations should be at the top of function blocks (only trumped by guards). Here, there is two ways to clean that up:
```javascript
telicon: function(pId, options) {
	var id = _.startsWith(pId, 'telicon2--')
			? pId
			: 'telicon2--' + pId,
		template = monster.ui.getSvgIconTemplate({
			id: id,
			attributes: options.hash
		});

	return new Handlebars.SafeString(template);
},
```
Or, more condensed:
```javascript
telicon: function(id, options) {
	return new Handlebars.SafeString(
		monster.ui.getSvgIconTemplate({
			id: _.startsWith(id, 'telicon2--')
				? id
				: 'telicon2--' + id,
			attributes: options.hash
		})
	);
},
```